src/main/java/com/splunk/kafka/connect/SplunkSinkConnectorConfig.java

```diff
@@ -405,4 +430,38 @@ private String getMetaForTopic(String[] metas, int expectedLength, int curIdx, S
         }
         return metaMap;
     }
+
+    protected static void validateKerberosConfigs(Map<String, Object> configs, ConfigValidationResult result) {
```

Comment:
need clarifications here...

Reply:
This is used for multi-config validation. If any of the Kerberos configs is set then all Kerberos configs must be set.